osf/admin.py

```diff
@@ -6,48 +6,33 @@
 
 class NodeAdmin(ForeignKeyAutocompleteAdmin):
     fields = (
-        'title',
-        'description',
+        'affiliated_institutions',
```

Comment:
This might be shorter: https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.ModelAdmin.exclude